### ISSUE TITLE
[UWP][Video Card not being rendered] 

### ISSUE THREAD

**Issue by Hammad-H** [microsoft/AdaptiveCards#4489]
# Platform
* UWP

# Version of SDK

Nuget 1.2.10

# Details
I am unable to render video cards. I have created a small app to test the issue. If you click the button it attempts to render the card ProductVideo.json from v1.0 cards. I get the following exception: 
![vidCardError](https://user-images.githubusercontent.com/33607998/88699527-7bb08980-d0d5-11ea-9679-e873d3ddcecd.PNG)

*This error does not occur when I test the card in the UWP visualizer.*

I have zipped my app(which contains the card as well) and have attached it with this bug
[TestVidCard.zip](https://github.com/microsoft/AdaptiveCards/files/4990027/TestVidCard.zip)

**Comment by paulcam206** [comment]
I see a repro as well... The same payload (after templating applied and everything) works fine in the visualizer against `main`. I'm looking to get a `release/1.2` repro to see if it happens there as well.

**Comment by paulcam206** [comment]
for reference, here's the literal payload (`cardPayload` in the screenshot):

```json
{"$schema":"http://adaptivecards.io/schemas/adaptive-card.json","type":"AdaptiveCard","version":"1.1","fallbackText":"This card requires Media to be viewed. Ask your platform to update to Adaptive Cards v1.1 for this and more!","body":[{"type":"Media","poster":"https://adaptivecards.io/content/poster-video.png","altText":"Adaptive Cards overview video","sources":[{"mimeType":"video/mp4","url":"https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"}]}],"actions":[{"type":"Action.OpenUrl","title":"Learn more","url":"https://adaptivecards.io"}]}
```

**Comment by paulcam206** [comment]
have repro in `release/1.2` visualizer. still digging, but this is a NULL AV in `XamlBuilder::WrapInTouchTarget` (`action` param is `nullptr`).

**Comment by Hammad-H** [comment]
The issue also occurs for v1.1/Tests/MediaAudioInColumnSet.json 
```
{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.0",
	"body": [
		{
			"type": "ColumnSet",
			"columns": [
				{
					"type": "Column",
					"items": [
						{
							"type": "TextBlock",
							"text": "This is a media element which specifies a poster:",
							"wrap": true
						},
						{
							"type": "Media",
							"poster": "http://adaptivecards.io/content/cats/1.png",
							"sources": [
								{
									"mimeType": "audio/mpeg",
									"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp3"
								}
							]
						}
					]
				},
				{
					"type": "Column",
					"items": [
						{
							"type": "TextBlock",
							"text": "This is a media element which doesn't specify a poster. It will use the poster from the host config if present:",
							"wrap": true
						},
						{
							"type": "Media",
							"sources": [
								{
									"mimeType": "audio/mpeg",
									"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp3"
								}
							]
						},
						{
							"type": "TextBlock",
							"text": "This text is here to check that we are not blocking the text below"
						}
					]
				}
			]
		}
	]
}

```
and v1.1/Tests/MediaInColumnSet.json 

```
{
	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
	"type": "AdaptiveCard",
	"version": "1.0",
	"body": [
		{
			"type": "ColumnSet",
			"columns": [
				{
					"type": "Column",
					"items": [
						{
							"type": "TextBlock",
							"text": "This is a media element which specifies a poster:",
							"wrap": true
						},
						{
							"type": "Media",
							"poster": "http://adaptivecards.io/content/cats/1.png",
							"sources": [
								{
									"mimeType": "video/mp4",
									"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"
								}
							]
						}
					]
				},
				{
					"type": "Column",
					"items": [
						{
							"type": "TextBlock",
							"text": "This is a media element which doesn't specify a poster. It will use the poster from the host config if present:",
							"wrap": true
						},
						{
							"type": "Media",
							"sources": [
								{
									"mimeType": "video/mp4",
									"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"
								}
							]
						}
					]
				}
			]
		}
	]
}

```

**Comment by paulcam206** [comment]
we're literally passing `nullptr` from `XamlBuilder::BuildMedia`. looks like this issue is fixed in `main` (along with some decent refactoring).

we just need to do a NULL check and default to a button here:
https://github.com/microsoft/AdaptiveCards/blob/d52b82fb5c96f6f57550a9c93fd2e633a50c940d/source/uwp/Renderer/lib/XamlBuilder.cpp#L4447-L4448

**Comment by paulcam206** [comment]
should be fixed when 1.2.11 ships
